uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -165,7 +164,7 @@ void testGetZone_nameRequiredError(String url) throws Exception {
     void testGetZone_nameEmptyError(String url) throws Exception {
         performMockMvcCallAndAssertError(get(url), status().isBadRequest(),
                                          JsonUtils.writeValueAsString(
-                                             new OrchestratorErrorResponse("getZone.name: must not be empty")),
+                                             new OrchestratorErrorResponse("getZone.name: must be specified")),
```

Comment:
also, here getZone in error message will create confusion between name and getZone.name.

Reply:
Done. Thanks!